tests/integration/docusaurus/connecting_to_your_data/how_to_configure_an_inferredassetdataconnector.py

```diff
@@ -185,12 +185,12 @@
     },
     "data_connectors": {
         "default_inferred_data_connector_name": {
-            "class_name": "InferredAssetFilesystemDataConnector",
+            "class_name": "InferredAssetS3DataConnector",
             "bucket": "<MY S3 BUCKET>/",
             "prefix": "<MY S3 BUCKET PREFIX>/",
             "default_regex": {
-                "group_names": ["data_asset_name", "year", "month"],
-                "pattern": "(.*)_(\d{4})-(\d{2})\.csv",
+                "group_names": ["prefix", "data_asset_name", "year", "month"],
+                "pattern": r"(.*)/(.*)_sample_(\d{4})-(\d{2})\.csv",
```

Comment:
From a discussion with @alexsherstinsky:

This adjustment to the regex was found to be necessary for InferredAssetS3DataConnector, which lists files differently than a typical FilesystemDataConnector.  The FilesystemDataConnector uses `glob` to list the files in the directory, and truncates the parent folders, while the InferredAssetS3DataConnector lists the files starting from the bucket-name. 

Since other cloud-DataConnectors list files the same way, it may be helpful to add a similar example to the how-to-guides for the other cloud-DataConnectors